x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -78,12 +78,8 @@ export const getSignalsTemplate = (index: string, spaceId: string, aadIndexAlias
             ...ecsMapping.mappings.properties.threat,
             properties: {
               ...ecsMapping.mappings.properties.threat.properties,
-              indicator: {
-                ...otherMapping.mappings.properties.threat.properties.indicator,
-                properties: {
-                  ...otherMapping.mappings.properties.threat.properties.indicator.properties,
-                  event: ecsMapping.mappings.properties.event,
-                },
+              enrichments: {
+                ...otherMapping.mappings.properties.threat.properties.enrichments,
```

Comment:
I was wondering why we are maintaining this piece separately from `ecsMapping` - shouldn't `enrichments` be reading from `ecsMapping` (instead of `otherMapping`) where `ecsMapping.mappings.properties.threat.properties.enrichments` is up to date with ECS 1.11?

Reply:
That's a good point; I had left these separate since they are still beta, but turns out that beta fields appear just like regular fields in the published mappings: https://github.com/elastic/ecs/blob/1.11/generated/elasticsearch/7/template.json so I agree that these should be in `ecsMapping` for consistency.

I had wanted to do a separate PR to update ALL mappings to 1.11 (which this PR doesn't do), so we can address this in that effort.